network.go

```diff
@@ -61,7 +61,7 @@ type NetworkController interface {
 	Networks() []Network
 
 	// WalkNetworks uses the provided function to walk the Network(s) managed by this controller.
-	WalkNetworks(walker NetworkWalker)
+	WalkNetworks(walker NetworkWalker) bool
```

Comment:
@squaremo @tomwilkie @aboch  I think there is a little misunderstanding here.

As I see it, the contract for the callback function `NetworkWalker` is to return a bool to break the walk routine `WalkNetworks`. This in no way indicating if the the walker callback is performing a search or any other functionality. Hence, returning that breaking condition from `NetworkWalker` doesnt seem appropriate and assume that it is used for searching. Hope that clarifies the reason for not returning the bool.

Having said that, I do see a need for `NetworkWalker` to return an error for actual error condition. But that doesn't apply to this case. In future, if there is an error such as unable to access the data, etc... occurs, it is good to return a error and inform the caller on the error condition.